middleware/i18n/i18n.go

```diff
@@ -43,15 +43,20 @@ func (t *Translator) Load() error {
 	return t.Box.Walk(func(path string, f packr.File) error {
 		b, err := t.Box.MustBytes(path)
 		if err != nil {
-			log.Fatal(err)
-			return errors.WithStack(err)
+			wErr := errors.Wrapf(err, "Unable to read locale file %s", path)
+			log.Fatal(wErr)
```

Comment:
I know this sounds silly, since there was already a `log.Fatal` in this code, but shouldn't we _not_ do that since it stops execution of the app? In theory the `return` on the next line never gets called.